lib/msal-browser/src/utils/PopupUtils.ts

```diff
@@ -82,6 +104,9 @@ export class PopupUtils {
         const left = Math.max(0, ((width / 2) - (BrowserConstants.POPUP_WIDTH / 2)) + winLeft);
```

Comment:
If the top/left are passed in to the popupConfig as zero then we should use the logic you have here to center the popup.

A very small thing to consider is that you shouldn't compute top/left unless it's needed.  So if width/height/top/left are all set, we should skip the math here.

Reply:
Done.